src/pages/collections/[id].tsx

```diff
@@ -105,13 +105,15 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     return { notFound: true };
   }
 
-  const import_id = context.params.id;
-  const apiClient = new ApiClient(process.env.CONCEPTS_API_URL);
-
   let collectionData: TCollectionPublicWithFamilies;
 
   try {
-    const { data: collectionResponse } = await apiClient.get(`/families/collections/${import_id}`);
+    const apiClient = new ApiClient(process.env.CONCEPTS_API_URL);
+    const id = context.params.id;
+    const { data: slugData } = await apiClient.get(`/families/slugs/${id}`);
```

Comment:
Side note: kind of weird we call `/families/slugs/` with the collection ID rather than the family ID but no blocker, just threw me off a bit